conanfile.py

```diff
@@ -4,14 +4,15 @@
 class Shar(ConanFile):
     settings        = "os",  "compiler",  "build_type"
 
-    requires        = ("asio/1.12.0@bincrafters/stable",             # network
+    requires        = ("asio/1.12.0@bincrafters/stable",              # network
                        "ScreenCaptureLite/16.1.0@0xd34d10cc/testing", # capture
                        "ffmpeg/4.0@0xd34d10cc/testing",               # encoder
                        "spdlog/0.17.0@bincrafters/stable",            # logs
                        "gtest/1.8.1@bincrafters/stable",              # UTs
                        # TODO: move to 0xd34d10cc
                        "prometheus-cpp/0.6.0@d34dpkgs/testing",       # metrics
-                       "jsonformoderncpp/3.5.0@vthiery/stable")       # config
+                       "jsonformoderncpp/3.5.0@vthiery/stable",       # config
```

Comment:
Seems like you forgot to add conan remotes to the appveyor and travis files, isn't it?

Reply:
Which one? All packages are either from conan-center or d34dpkgs.